Dockerfile

```
@@ -0,0 +1,5 @@
+FROM python:alpine
+RUN python3 -m pip install -U gallery-dl yt-dlp
+RUN apk update
+RUN apk add ffmpeg
+ENTRYPOINT [ "gallery-dl" ]
```

```suggestion
ENTRYPOINT [ "gallery-dl" ]

```
Please add a line break to the end of the file